src/core/core.element.js

```diff
@@ -80,7 +80,7 @@ helpers.extend(Element.prototype, {
 
 		// No animation -> No Transition
 		if (!model || ease === 1) {
-			me._view = model;
+			me._view = helpers.clone(model);
```

Comment:
It's confusing to me that in order to make animation work you had to change code that has a comment that says "No animation". Is the comment wrong?

Reply:
This block actually terminates the current transition, but this change makes sure that `_view` and `_model` points to different objects with the equivalent values. When the next animation starts, `_view` is cloned to `_start`, and values can be transitioned between `_start.*` and a new `_model.*` value. Without the fix, `_start.*` would be the same value as `_model.*`.